localnet/with-ibc/stop.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+echo "Stop chains"
+$SCRIPTDIR/finschia/stop.sh
+$SCRIPTDIR/finschia2/stop.sh
```

Comment:
Why do you write this command again instead of using `./script/stop-chains.sh`?
and where is the logic to halt relayers?